app/src/main/res/values/strings.xml

```diff
@@ -435,6 +435,7 @@
   <string name="new_hint_available">New hint available</string>
   <string name="no_new_hint_available">No new hint available</string>
   <string name="show_hints_and_solution">Show hints and solution</string>
+  <string name="hint">Hint %s</string>
```

Comment:
This should be named in-context ("hint" by itself is a rather vague and potentially confusing string name).